.github/workflows/main.yml

```diff
@@ -459,12 +552,13 @@ jobs:
     - uses: actions/checkout@v4
     - run: rustup update --no-self-update 1.76 && rustup default 1.76
     - run: cargo build
-      
+
 
   deploy:
     permissions:
       contents: write #  to push changes in repo (jamesives/github-pages-deploy-action)
 
+    if: github.repository == 'rustwasm/wasm-bindgen'
```

Comment:
Why?
This didn't make any difference in the CI time.

Reply:
It fails when running in other repositories: https://github.com/Systemcluster/wasm-bindgen/actions/runs/10274428735/job/28431248685

Reply:
Ah, you are checking the repo, apologies, didn't read carefully enough.